specs/chain/hs/src/Cardano/Spec/Chain/STS/Rule/Chain.hs

```diff
@@ -181,6 +181,8 @@ instance HasTrace CHAIN where
     ct <- Gen.integral (Range.linear 1 7)
     -- Update adoption threshold
     uat <- Gen.integral (Range.linear 1 7)
+    -- Chain stability
```

Comment:
Do we want to make this different from `2*k`? We might have to restrict this when testing the actual validation code.

Reply:
But of course, we can make this once we change the spec, if we do want to make this.